django/db/backends/postgresql/introspection.py

```diff
@@ -113,7 +113,8 @@ def get_table_description(self, cursor, table_name):
             FieldInfo(
                 line.name,
                 line.type_code,
-                line.display_size,
+                # display_size is always None on psycopg2.
+                line.internal_size if line.display_size is None else line.display_size,
```

Comment:
won't this condition always evaluate to `True`?

Reply:
`display_size` is filled properly on `psycopg`, see ticket-33308.